modules.json

```diff
@@ -3,14 +3,20 @@
     "homePage": "https://github.com/nf-core/airrflow",
     "repos": {
         "nf-core/modules": {
-            "custom/dumpsoftwareversions": {
-                "git_sha": "e745e167c1020928ef20ea1397b6b4d230681b4d"
-            },
-            "fastqc": {
-                "git_sha": "49b18b1639f4f7104187058866a8fab33332bdfe"
-            },
-            "multiqc": {
-                "git_sha": "49b18b1639f4f7104187058866a8fab33332bdfe"
+            "git_url": "https://github.com/nf-core/modules.git",
+            "modules": {
```

Comment:
Is it correct there are only 3 modules?

Reply:
yes, all other modules are local. Need to add them to `nf-core/modules` at some point but unlikely that other pipelines will use them as they are very TCR / BCR specific tools...